addons/web/static/tests/views/form/form_view_tests.js

```diff
@@ -13265,4 +13265,57 @@ QUnit.module("Views", (hooks) => {
             "this is full personalize setting"
         );
     });
+
+    QUnit.test("quit form view when save actually deletes", async function (assert) {
+        const formView = registry.category("views").get("form");
+        class CustomFormController extends formView.Controller {
```

Comment:
why do you need this? We would like to test what happens with the "real" form controller

Reply:
Nothing will happen in the main controller, as `saveButtonClicked` will be called without the arg `throwOnError`.
The idea here is to be able to override `saveButtonClicked` in a specific module to be able to call super with this arg and, if needed, catch the error

Reply:
I would prefer you to test your custom controller then, in your addon. Here the test is too artificial, and very likely to become useless in the future when we/you do refactorings.

Reply:
So, if I understand correctly: no test for this commit, right?

Reply:
Yes, a test, but in your addon, using the custom view that does the throwOnError stuff (I don't like this btw)

Reply:
@aab-odoo Done. See test here https://github.com/odoo/enterprise/pull/31170/files#diff-aa495db0c4f292f130f9fc27e4c80252c5a747bd5c91133246a4307eed250d07R1-R71

Reply:
Why don't you add it in a branch with same name as this one, s.t. they are both merged (and linked together if someone needs to debug later)?

Reply:
test is ok though